episodes/chap4_manage_via_ui.md

```diff
@@ -150,8 +152,9 @@ You will need the API keys when you run the nodes to authenticate with the vanta
 Go to the administration page in the vantage6 UI, and do the following:
 
 1. Create a new organization
-2. Create a new user
-3. Create a new collaboration
+2. Create a new user (`Researcher` role plus the permission to view his own task)
+3. Check what permissions you have, can you create a new user and grant him the permissions you don't have?
```

Comment:
Nitpicking: you are using `him` and `his` -> can you make it `them` and `theirs`?